applications_tests/gls_nitsche_navier_stokes_33/sphere.geo

```diff
@@ -0,0 +1,5 @@
+SetFactory("OpenCASCADE");
+Sphere(1) = {0, 0, 0, 0.5, -Pi/2, Pi/2, 2*Pi};
+Physical Surface(0) = {1};
+Physical Volume(0) = {1};
+Mesh.MeshSizeFactor = 2;
```

Comment:
Have you checked that this is well taken into account during the mesh generation? To my experience with GMSH, mesh control options can be discarded by OpenCascade. This would not be troublesome here, but if you need to have a higher control over your mesh you may have to switch to the default generator. Just keep that in mind :)

Reply:
I checked when making the geo file, but I'll double check.

Reply:
With the latest versions of gmsh there is no problem